README.md

```diff
@@ -82,7 +82,8 @@ feature themselves.
 
 Run `yarn` to install the necessary dependencies, and run `yarn start` to start
 the automatic builder. Then load `examples/app-menu/` as an unpacked extension
-into Google Chrome.
+into Google Chrome. You may need to manually copy `background.js` from
+`packages/core/` to the `examples/app-menu/` directory.
```

Comment:
This shouldn't be necessary. The `--examples` flag passed to gulp should handle this. To test:

```sh
yarn gulp clean
ls examples/app-menu/background.js # this file should not exist
yarn gulp --examples               # yarn start includes this
ls examples/app-menu/background.js # this file should now exist
```

```suggestion
into Google Chrome.
```

Reply:
If the file isn't being generated for you, what node version and OS are you using?

Reply:
https://github.com/InboxSDK/InboxSDK/pull/1188 should hopefully patch up the issue for you.

Reply:
Addressed by #1192.